config.w32

```diff
@@ -224,6 +224,52 @@ if (PHP_MONGODB != "no") {
     WARNING("mongodb libsasl support not enabled, libs not found");
   }
 
+  if (PHP_CLIENT_SIDE_ENCRYPTION != "no" && mongoc_ssl_found) {
+    mongoc_opts.MONGOC_ENABLE_CLIENT_SIDE_ENCRYPTION = 1;
+
+    PHP_MONGODB_CFLAGS += "\
+      /I" + configure_module_dirname + "/src/libmongocrypt/src \
+      /I" + configure_module_dirname + "/src/libmongocrypt/kms-message/src \
+      /I" + configure_module_dirname + "/src/libmongocrypt-compat \
+    ";
+
+    var mongocrypt_opts = {
+      MONGOCRYPT_ENABLE_TRACE: 1,
+      MONGOCRYPT_IS_POSIX: 0,
+      MONGOCRYPT_IS_WIN: 1,
+      MONGOCRYPT_ENABLE_CRYPTO: 0,
+      MONGOCRYPT_ENABLE_CRYPTO_LIBCRYPTO: 0,
```

Comment:
`CRYPTO` and `CRYPTO_LIBCRYPTO` can be defined to `1` here, since we know we'll be using OpenSSL at this point. This may address your earlier comment where you speculated that Windows was building with no crypto.

Reply:
That is correct. We'll have to revise this when adding support for Secure Channel on Windows, but until then this works fine.